src/main/java/com/example/cookbook/recipe/Recipe.java

```diff
@@ -0,0 +1,149 @@
+package com.example.cookbook.recipe;
+
+import com.example.cookbook.category.Category;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+public class Recipe {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+    private String recipeImgUrl;
+
+    private Integer prepareTime;
+
+    private String description;
+
+    private String ingredients;
+
+    private String instruction;
+
+    private int likedRecipe;
+
+    @DateTimeFormat(pattern = "yyyy-MM-dd")
+    private LocalDate date;
+
+    @ManyToOne
+    @JsonIgnore
```

Comment:
To jest adnotacja specyficzna dla kontrolerów, nie powinno jej być w warstwie dostępu do danych / encjach